recipesearch/src/Recipes/reducer.js

```diff
@@ -1,8 +1,12 @@
 import { combineReducers } from 'redux'
 import  searchBarReducer from './SearchBar/reducer'
+import searchIngredientsReducer from './SearchIngredients/reducer';
```

Comment:
Make changes that make it possible to alter this import statement to
```JavaScript
import { searchIngredientsReducer } from './SearchIngredients';
```

Reply:
Let me work on this